sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/DoFnTester.java

```diff
@@ -654,6 +687,41 @@ public Void dispatch(DoFnSignature.Parameter.WindowParameter p) {
               return null;
             }
 
+            @Override
+            @Nullable
+            public Void dispatch(DoFnSignature.Parameter.ElementParameter p) {
+              return null;
```

Comment:
I don't think any of these should be null or nullable, but should throw.

They are also totally not necessary, since `WithDefault` means you don't have to override any case you are not interested in.